__tests__/app.test.js

```diff
@@ -211,23 +241,48 @@ describe("GET /api/articles/:article_id/comments", () => {
       .expect(200)
       .then((res) => {
         expect(res.body.comments).toEqual([
-      {
-        comment_id: 1,
-        body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
-        article_id: 9,
-        author: 'butter_bridge',
-        votes: 16,
-        created_at: '2020-04-06T12:17:00.000Z'
-      },
-      {
-        comment_id: 17,
-        body: 'The owls are not what they seem.',
-        article_id: 9,
-        author: 'icellusedkars',
-        votes: 20,
-        created_at: '2020-03-14T17:02:00.000Z'
-      }
-    ]);
+          {
+            comment_id: 1,
+            body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+            article_id: 9,
+            author: "butter_bridge",
+            votes: 16,
+            created_at: "2020-04-06T12:17:00.000Z",
+          },
+          {
+            comment_id: 17,
+            body: "The owls are not what they seem.",
+            article_id: 9,
+            author: "icellusedkars",
+            votes: 20,
+            created_at: "2020-03-14T17:02:00.000Z",
+          },
+        ]);
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
```

Comment:
Error handling for the post request?

Reply:
error handling for the post request is all in the error handling describe block